ipaclient/install/client.py

```diff
@@ -2310,8 +2310,9 @@ def install_check(options):
 
     # Check if old certificate exist and show warning
     if (
-        not options.ca_cert_file and
-        get_cert_path(options.ca_cert_file) == paths.IPA_CA_CRT
+        not options.ca_cert_file
+        and get_cert_path(options.ca_cert_file) == paths.IPA_CA_CRT
+        and os.stat(paths.IPA_CA_CRT).st_size != 0
```

Comment:
This change doesn't fix the issue.
If there is a file /etc/ipa/ca.crt but it is empty and ipa-client-install does not specify the --ca-cert-file option: options.ca_cert_file = None
`not options.ca_cert_file` is True
`get_cert_path(None)` returns /etc/ipa/ca.crt
the size check returns == 0
=> the warning is not printed but it doesn't change the logic, get_cert_path still returns /etc/ipa/ca.crt and the install fails.
The change should be done in get_cert_path:
```
    if cert_path is not None:
        return cert_path

    if os.path.exists(paths.IPA_CA_CRT) and os.stat(paths.IPA_CA_CRT).st_size != 0:
        return paths.IPA_CA_CRT

    return None
```